php/class-wp-seo.php

```diff
@@ -540,7 +555,35 @@ public function wp_head() {
 			} else {
 				$key = false;
 			}
+			return $key;
+		}
+
+		/**
+		 * Determine the <meta> values for the current page.
+		 *
+		 * Unlike WP_SEO::wp_title(), custom per-entry and per-term values are not
+		 * returned immediately but rendered at the end of the method.
+		 *
+		 * @see WP_SEO::meta_field() for detail on how the values are rendered.
+		 */
+		public function wp_head() {
+			$key = $this->get_key();
 
+			if ( is_singular() ) {
+				if ( WP_SEO_Settings()->has_post_fields( $post_type = get_post_type() ) ) {
+					$meta_description = $this->format( get_post_meta( get_the_ID(), '_meta_description', true ) );
```

Comment:
@joshkadis Think it is too late for this one since we have the pattern out already.